conf/messages.en

```diff
@@ -61,6 +61,17 @@ invalidRequest.li2 = Make a declaration if you are:
 invalidRequest.Import.restart = bringing commercial goods into Great Britain
 invalidRequest.Export.restart = taking commercial goods out of Great Britain
 
+# DeclarationNotFound
+declarationNotFound.title = Are you bringing in goods from EU countries, non-EU countries or both?
+declarationNotFound.heading = Declaration not found
```

Comment:
title and heading should generally be the same, please double check with Ben

Reply:
Both bits of text are cut/paste from the prototype, but can double check with Ben

Reply:
The prototype titles are never right, I have raised it previously but it's fallen on deaf ears

Reply:
@jordanrowe do you still want me to check with Ben? or just make them match

Reply:
Make them match, but I'll bring it up in standup today. Will be more vigilant in refinements as well, when we are going through the prototype I'll try and stay awake and call out these types of things